web/client/plugins/manager/GroupManager.jsx

```diff
@@ -88,8 +95,93 @@ class GroupManager extends React.Component {
  * Allows an administrator to browse user groups.
  * Renders in {@link #plugins.Manager|Manager} plugin.
  * @name GroupManager
+ * @property {object[]} [attributeFields] attributes that should be shown in attributes tab of group manager.
+ * @property {boolean} [showMembersTab=true] shows/hides group members tab
+ * @property {boolean} [showAttributesTab=false] shows/hides group attributes tab
  * @memberof plugins
  * @class
+ * @example
+ * { "name": "GroupManager",
+ *   "cfg": {
+ *        "showMembersTab": false,
+ *        "showAttributesTab": true,
+ *        "attributeFields": [
+ *             {
+ *                 "title": "Simple text",
```

Comment:
**NOTE:** 
for the future we may want to localize these labels, by using paths to translations ids and using Message comp in the UserDialog comp or GroupDialog comp

Reply:
yes, Good suggestion. If it will be required in the future, we can do a separate PR.